### PR TITLE
[noissue] MiniCPM-o 4.5 Ascend qualification implementation

### PR DESCRIPTION
**Pull request by xiaojiu1414**
Team: noissue

This PR contains the reproducible MiniCPM-o 4.5 Ascend implementation
and official accuracy evidence for the competition qualification.

### CODE REVIEW

File: tests/model_executor/stage_input_processors/test_minicpmo_4_5_omni.py [hunk 1]
```diff
@@ -3,8 +3,10 @@
 import pytest
 import torch
 
+import vllm_omni.model_executor.stage_input_processors.minicpmo_4_5_omni as minicpmo_module
 from vllm_omni.model_executor.stage_input_processors.minicpmo_4_5_omni import (
     _extract_first_audio_ref,
+    _trace_ref_audio,
```
Comment (lishunyang12) [inline tests/model_executor/stage_input_processors/test_minicpmo_4_5_omni.py:9]: Remove the _trace_ref_audio import and test; that helper doesn't exist.

File: vllm_omni/entrypoints/openai/serving_chat.py [hunk 2]
```diff
@@ -203,6 +203,83 @@ def _has_minicpmo45_stage(self) -> bool:
...
+            for part in content:
+                if not isinstance(part, dict) or part.get("type") not in {
+                    "audio_url",
+                    "input_audio",
+                    "audio",
+                }:
+                    continue
+                audio = part.get("audio_url", part.get("input_audio", part.get("audio")))
+                if isinstance(audio, dict):
+                    audio = audio.get("url") or audio.get("data")
```
Comment (lishunyang12) [inline vllm_omni/entrypoints/openai/serving_chat.py:244]: Wrap input_audio data in a data URL using its format.

File: vllm_omni/model_executor/models/minicpmo_4_5/minicpmo_4_5_omni_tts.py [hunk 3]
```diff
@@ -44,20 +47,131 @@
...
+    ``condition_tokens * 10`` cap is not part of that contract and can cut off
+    perfectly valid Chinese utterances before EOS; ``condition_tokens`` is
+    retained for API/test compatibility and intentionally unused.
     """
-    return max(
-        _MIN_AUDIO_TOKENS,
-        min(_MAX_AUDIO_TOKENS, condition_tokens * _AUDIO_TOKENS_PER_TEXT_TOKEN),
-    )
+    del condition_tokens
+    return _MAX_AUDIO_TOKENS
```
Comment (lishunyang12) [inline vllm_omni/model_executor/models/minicpmo_4_5/minicpmo_4_5_omni_tts.py:174]: Update the remaining max_tokens assertions to match the new 2048 default.
